### PR TITLE
Fixes baseturf helpers

### LINKED ISSUE #41147: Lavaland lava and surface turfs placed by ruin/terrian generation are duplicating their baseturf lists
Master

image
image
image

### LINKED ISSUE #79172: Baseturfs on closed turfs are fucky wucky, Produces runtime.
## Reproduction:
1. Put a baseturf on a room with walls
2. Launch ss13, load the room and drop a bomb on it
3. watch as the open turfs correctly gets destroyed to the baseturf while the closed turfs runtimes
image

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Baseturf helpers do this:
image

And they also do this (up to 3x):
Screenshot 2023-11-12 164334

When trying to debug `PlaceOnBottom` I couldn't help but notice it's trying to encompass use cases that simply aren't used anywhere. YAGNI - this makes it much more complex than it needs to be. 
## Why It's Good For The Game
Fixes weird baseturf handling
FIxes #79172
## Changelog
N/A nothing player facing

### CODE REVIEW

**Comment by jlsnow301** [comment]
This solution just seems to deceptively simple that I feel I'm overlooking things, reviews are welcome here

File: code/game/turfs/baseturfs.dm [hunk 1]
```diff
@@ -20,27 +20,17 @@
...
-	baseturfs = baseturfs_string_list(new_baseturfs + baseturfs, src)
+/// Places the given turf on the bottom of the turf stack.
+/turf/proc/place_on_bottom(turf/bottom_turf)
+	var/list/turf/new_turfs = list()
+
+	new_turfs.Add(
+		initial(bottom_turf.baseturfs),
+		bottom_turf,
+		baseturfs,
+	)
```
Comment (Mothblocks) [inline code/game/turfs/baseturfs.dm:31]: Inline the list instead

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by Singul0** [comment]
god bless you 🙏  I can start working on my syndicate waystation assault map again

**Comment by san7890** [comment]
the unit test didn't break so i'll trust this works

**Pull request merged**